action.yml

```diff
@@ -8,15 +8,15 @@ inputs:
   source_gh_token:
     description: 'GitHub Token for the repo to be synced from. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
     required: false
-    default: ""
+    default: ${{ github.token }}
   target_gh_token:
     description: 'GitHub Token for the repo to be synced from. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}'
```

Comment:
I think we have to adjust the description.  "GitHub Token for the repo to be synced to. Can be passed in using $\{{ secrets.GITHUB_TOKEN }}"

Reply:
Yes. Copy paste issue

- [ ] update docs